src/pds/registrysweepers/legacy_registry_sync/legacy_registry_sync.py

```diff
@@ -65,5 +65,5 @@ def run(
         if not ok:
```

Comment:
[suggest] if `ok` is a boolean, make the label slightly more descriptive like `result_is_ok` (or whatever is most-accurate).  If `ok` is a non-boolean and is being coerced from a string, int etc, perform an explicit comparison based on the expected condition(s) rather than coercing for implicit comparison.

Reply:
Tanks @alexdunnjpl , I made that update.